src/components/SearchForm/SearchForm.jsx

```diff
@@ -13,11 +30,13 @@ export default function SearchForm() {
                     personal account.
                 </h2>
             </div>
-            <form className='search-form__search'>
+            <form className='search-form__search' onSubmit={handleSubmit}>
                 <input
                     className='search-form__search-text'
                     type='text'
                     placeholder='Enter topic'
+                    name='keyword'
+                    onChange={handleChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components